src/components/message-input/index.tsx

```diff
@@ -290,78 +294,115 @@ export class MessageInput extends React.Component<Properties, State> {
   };
 
   renderInput() {
+    const hasInputValue = this.state.value?.length > 0;
+
+    console.log(this.props.isMessengerFullScreen);
```

Comment:
debugging log? I am not alone it seems :joy:

Reply:
Haha, always my friend ! 

Removed.